Dockerfile

```
@@ -41,7 +41,7 @@ COPY --from=js-static $APP_DIR/bundles ./bundles
 
 RUN ./manage.py collectstatic
 
-FROM amazon/aws-lambda-python:3.8
+FROM amazon/aws-lambda-python:3.11.8
```

3.11.8 is not a tag AWS publishes https://hub.docker.com/r/amazon/aws-lambda-python/tags?page=&page_size=&name=3.11&ordering=last_updated. Python [3.11.9](https://pythoninsider.blogspot.com/2024/04/python-3119-is-now-available.html) (which caused this bug) was released april 8th

[3.11.2024.03.28.16](https://hub.docker.com/layers/amazon/aws-lambda-python/3.11.2024.03.28.16/images/sha256-0e24f7bb734e2d668406e08be78b38bb3c08522cd94c242e8275f4a8533ec410?context=explore) is the tag from before 3.11.9 launch
```suggestion
FROM amazon/aws-lambda-python:3.11.2024.03.28.16
```